apollo.config.js

```diff
@@ -0,0 +1,23 @@
+require('dotenv').config();
+
+// Configuration for the vscode-apollo extension.
+module.exports = {
+  client: {
+    service: {
+      name: 'are.na',
+      url: 'https://api.are.na/graphql',
+      headers: {
+        'X-APP-TOKEN': process.env.GRAPHQL_XAPP_TOKEN,
```

Comment:
This is where apollo's tooling pulls in the schema. I suspect the issue with `@client` is that it's pulling a remote schema which has no notion of `@client` and then failing to recognize client-only additions. I'll need to play with it a bit more.